tests/test_s3crypt.py

```diff
@@ -44,3 +46,75 @@ def test_init(self):
         # Run a second time to ensure it is idempotent
         s3crypt.init()
         self.assertEqual(s3crypt.config, self.config_data)
+
+    def test_s3_event_empty(self):
+        s3crypt.init()
```

Comment:
rather than repeat you could just do as a setUp method, also given empty variants its a little  unclear. also note your modifying module global state with init without cleaning up after at the moment.

Reply:
Got rid of those repeats as the tests were effectively useless.